packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx

```diff
@@ -0,0 +1,146 @@
+                        <a
+                            href={`${PATH_PROCESSES}/${subscriptionDetailProcess.processId}`}
+                        >
+                            {subscriptionDetailProcess.processId}
+                        </a>
+                    </td>
+                    <td css={emptyCellStyle}></td>
+                </tr>
+                <tr>
+                    <td css={emptyCellStyle}></td>
+                    <td css={headerCellStyle}>{t('status')}</td>
+                    <td css={contentCellStyle}>
+                        <WFOProcessStatusBadge
+                            processStatus={subscriptionDetailProcess.lastStatus}
+                        />
+                    </td>
+                    <td css={emptyCellStyle}></td>
+                </tr>
+                <tr>
+                    <td css={emptyCellStyle}></td>
+                    <td css={headerCellStyle}>{t('startedAt')}</td>
+                    <td css={contentCellStyle}>
+                        {parseDateToLocaleDateTimeString(
+                            parseDate(subscriptionDetailProcess.startedAt),
+                        )}
+                    </td>
+                    <td css={emptyCellStyle}></td>
+                </tr>
+                <tr>
+                    <td css={emptyCellStyle}></td>
+                    <td css={lastHeaderCellStyle}>{t('startedBy')}</td>
+                    <td css={lastContentCellStyle}>
+                        {subscriptionDetailProcess.createdBy}
+                    </td>
+                    <td css={emptyCellStyle}></td>
+                </tr>
+            </table>
+        </div>
+    );
+};
+
+interface WfoRenderSubscriptionProcess {
+    subscriptionDetailProcess: SubscriptionDetailProcess;
+}
+const WfoRenderSubscriptionProcess = ({
+    subscriptionDetailProcess,
+}: WfoRenderSubscriptionProcess) => {
+    const { timeLineStyle, workflowTargetStyle } =
+        useWithOrchestratorTheme(getStyles);
+
+    return (
+        <EuiComment
+            username={subscriptionDetailProcess.workflowTarget ?? ''}
+            timelineAvatarAriaLabel={subscriptionDetailProcess.workflowName}
+            timelineAvatar={
+                <EuiAvatar name={subscriptionDetailProcess.workflowTarget} />
+            }
+        >
+            <div css={timeLineStyle}>
+                <EuiText css={workflowTargetStyle}>
+                    {upperCaseFirstChar(
+                        subscriptionDetailProcess.workflowTarget,
+                    )}
+                </EuiText>
+                <EuiText>{subscriptionDetailProcess.workflowName}</EuiText>
+            </div>
+
+            <WfoProcessCard
+                subscriptionDetailProcess={subscriptionDetailProcess}
+            />
+        </EuiComment>
+    );
+};
+
+interface WfoProcessesTimelineProps {
+    subscriptionDetailProcesses: SubscriptionDetailProcess[];
+}
+
+export const WfoProcessesTimeline = ({
```

Comment:
New style naming

Reply:
+1 right?

Reply:
I was commenting about the inconsistency. But up to you if you do something with it: when we no do "new" code in the "new" style it will be, slightly, harder to do the big rename. I am ok with doing new components in the new style.

Just something to be aware of.

Reply:
There is another ticket to do the bulk but I thought it was good to start using the new style. I'm not sure it makes it that much harder but I can pick up the ticket next so I have to deal with it